hunt/app/views/animals/new.html.erb

```diff
@@ -1,2 +1,24 @@
-<h1>Animals#new</h1>
-<p>Find me in app/views/animals/new.html.erb</p>
+
+<div class="container">
+  <div class="row">
+    <div class="col">
+  <div class="card">
+    <div class="card-header">Create an Animal</div>
+    <div class="card-body">
+
+      <%= simple_form_for @animal do |f| %>
+        <%= f.error_notification %>
+
+        <%= f.input :name, required: true, error: "Name can't be blank" %>
+        <%= f.input :state, collection:$STATES, required: true, error: "State can't be blank"%>
+        <%= f.input :category, collection: $CATEGORIES, required: true, error: "Category can't be blank"%>
+
+
+        <button class='btn btn-dark'>Create an Animal<% f.button :submit %></button>
```

Comment:
Similar to `link_to` you can add a class to the submit button as well:
`<%= f.submit class: 'btn btn-dark' %>`